pkg/cli/admin/release/extract_tools.go

```diff
@@ -489,10 +492,19 @@ func (o *ExtractOptions) extractCommand(command string) error {
 
 		// copy the input to disk
 		if target.InjectReleaseImage {
+			releaseInfo := exactReleaseImage
 			var matched bool
-			matched, err = copyAndReplaceReleaseImage(w, r, 4*1024, exactReleaseImage)
+			errmsg := ""
+			switch target.Command {
```

Comment:
Honestly, this switch could be reversed since only installer relies on full release info and everything else (`default`) should be using the other, but that can be fixed later on when we'll be extending this mechanism at some point.